### ISSUE TITLE
Balancing suggestion -- Mob Heads

### ISSUE THREAD

**Issue by BBoldt** [squeek502/VeganOption#16]
Paper maché heads are good in theory, but I feel like things made from materials such as that should be mostly decorative. Dyes are much too easy to obtain considering what can be done with skulls in most mods. They are generally used for special recipes and meant to be a bit rare. 

So, I suggest maybe adding some sort of in-world mechanic to get skulls. Maybe something that requires sacrificing a certain amount of materials into some sort of construct.

For instance, consider I want an enderman head. I would make the paper mache mob head base, and throw it into the construct along with some ender pearls and some sort of catalyst item to "activate" the transformation. Out pops an enderman head.

**Comment by squeek502** [comment]
Yeah, it's not very well balanced currently (especially wither skulls). I originally thought melon blocks were gated behind silk touch, but unfortunately there's a vanilla recipe that turns 9 slices into a melon block, so that's out the window.

It seems like consuming experience to 'infuse' the mob heads would be fitting, but that would likely just encourage mob grinders...

Definitely something I want to address though. Need to think about it some more.

**Comment by BBoldt** [comment]
By the way, I'm using this mod in a large modpack on hard mode. I consider most your recipes for things to be balanced enough to be enabled by default. You've done a great job making alternative ways of getting things for people who don't like killing mobs (and even for people that do).

**Comment by squeek502** [comment]
Cool, I haven't received much feedback about balance, so that's nice to hear.

**Comment by BBoldt** [comment]
I love all the unique in world crafting mechanics. Forcing players to set up interesting systems if they want to automate them. Quite a bit more involved and interesting than most "mob drop crop" mods.
